spec/acceptance/foreman_cli_plugins_spec.rb

```diff
@@ -46,6 +48,12 @@ class { 'foreman::cli':
         it { is_expected.to be_installed }
       end
     end
+
+    if fact('os.family') == 'RedHat'
```

Comment:
This is missing the actual `describe package(package_name)` part?

Reply:
Fixed